tasks/get_current_archive_list/schemas/input.json

```diff
@@ -4,63 +4,50 @@
   "title": "get_current_archive_list Lambda Config",
   "description": "The input for the get_current_archive_list Lambda.",
   "type": "object",
-  "properties": {
-    "Records": {
-      "type": "array",
-      "description": "A list of dicts from SQS. From example: https://www.juniper.net/documentation/us/en/software/jsa7.4.2/jsa-dsm/topics/concept/jsa-dsm-create-sqs-queue-and-configure-s3-objectcreated-notifications.html",
-      "items": {
-        "description": "An s3 event passed in by SQS.",
+    "required": [
+      "awsRegion",
+      "s3"
+    ],
+    "properties": {
+      "awsRegion": {
+        "type": "string",
+        "description": "The region containing the bucket the event occurred in."
+      },
+      "s3": {
```

Comment:
This schema feels like this could be simplified and flattened to just the properties:
- `awsRegion`
- `bucket` - Possibly rename to `orcaBucket`
- `key`

I feel we can remove `s3` and `object` objects that are nested and do not add any value. This will require minor changes in the lambda.

Reply:
This was made to match the s3 event format, but now that we're going to have another lambda reading those we could translate there if that sounds good. Alternatively, if we just want that lambda to pass along the event and do all processing here, we should leave this as is. Not sure which is cleaner.

Reply:
Build translation into intermediary lambda.